Test Case 1 Input:
"1""2""3""4""5"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 2 Input:
"1" "2" "3""4" "5" "6""7" "8" "9""3""1" "2" "3"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 3 Input:
"32-202"Test Case Input:"-24-305"Test Case Input:"53-2042"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 4 Input:
"3" "2""-2""0""2"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 5 Input:
"3" 2 -2 0 2
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 6 Input:
"3" (String)"2" (String)"-2" (String)"0" (String)"2" (String)
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 7 Input:
"3" (String)2-202
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 8 Input:
"3" 2 -2 0 2
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 9 Input:
"3.5""2""-2""0""2"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 10 Input:
"3\n2\n-2\n0\n2"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 11 Input:
"5""3""-2""0""4""2"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 12 Input:
"5" "3" "-2" "0" "4" "2"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 13 Input:
"a""b""c""d""e"
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 14 Input:
"abc"2-202
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 15 Input:
"abc"21-305
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 16 Input:
-1
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 0
	at Main.main(Main.java:15)
Test Case 17 Input:
-12-305
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 0
	at Main.main(Main.java:15)
Test Case 18 Input:
-10Test Case Input:51 0Test Case Input:31 nullTest Case Input:41 22 33 45 6Test Case Input:21 23 4
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -9
	at Main.main(Main.java:6)
Test Case 19 Input:
-20-305
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -1
	at Main.main(Main.java:6)
Test Case 20 Input:
-24-305
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -1
	at Main.main(Main.java:6)
Test Case 21 Input:
-24005
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -1
	at Main.main(Main.java:6)
Test Case 22 Input:
-3
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:6)
Test Case 23 Input:
-30-252
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:6)
Test Case 24 Input:
-301-23
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:6)
Test Case 25 Input:
-302-24
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:6)
Test Case 26 Input:
-31 22 3
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -2
	at Main.main(Main.java:6)
Test Case 27 Input:
-5
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -4
	at Main.main(Main.java:6)
Test Case 28 Input:
-5Test Case Input:101 a2 b3 c4 d5 e6 f7 g8 h9 i10 jTest Case Input:31 22 3Test Case Input:410 2020 3030 4040 50Test Case Input:51 22 33 44 55 6
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -4
	at Main.main(Main.java:6)
Test Case 29 Input:
-51 21 33 43 5
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -4
	at Main.main(Main.java:6)
Test Case 30 Input:
-52 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Error:
Exception in thread "main" java.lang.NegativeArraySizeException: -4
	at Main.main(Main.java:6)
Test Case 31 Input:
0
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 32 Input:
0Test Case Input:31 22 3Test Case Input:41 22 33 4Test Case Input:51 21 33 43 5Test Case Input:61 21 32 42 54 6
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 33 Input:
0Test Case Input:31 22 33 4Test Case Input:51 22 33 44 55 6
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 34 Input:
00000
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 35 Input:
01 22 33 44 5
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 36 Input:
01 22 33 44 5Test Case Input:31 22 33 44 5Test Case Input:51 22 33 44 55 6Test Case Input:61 22 33 44 55 66 7
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 37 Input:
01 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 251 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 251 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 38 Input:
01 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 39 Input:
02468
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 40 Input:
02 33 44 55 6
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 41 Input:
02 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 42 Input:
03Test Case Input:501234Test Case Input:33123Test Case Input:3212-3Test Case Input:32123
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 43 Input:
0 0 00 0 00 0 031 2 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 44 Input:
0 11 22 33 44 5
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at Main.main(Main.java:15)
Test Case 45 Input:
10-1
Output:
Snuke
Test Case 46 Input:
11-503
Output:
Snuke
Test Case 47 Input:
110
Output:
Snuke
Test Case 48 Input:
1111 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 11 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
Output:
Snuke
Test Case 49 Input:
111 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 11 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1
Output:
Snuke
Test Case 50 Input:
11a1
Output:
Snuke
Test Case 51 Input:
11null
Output:
Snuke
Test Case 52 Input:
11 2
Output:
Snuke
Test Case 53 Input:
12345
Output:
Snuke
Test Case 54 Input:
1234567893123
Output:
Snuke
Test Case 55 Input:
123456789512345
Output:
Snuke
Test Case 56 Input:
1Test Case Input:21 2Test Case Input:31 22 3Test Case Input:41 22 33 4Test Case Input:51 21 33 43 5
Output:
Snuke
Test Case 57 Input:
1a b c d e f g h i j k l m n o p q r s t u v w x y z1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 261 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 261 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 261 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 261 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 261 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Output:
Snuke
Test Case 58 Input:
1 2 34 5 67 8 901 2 3
Output:
Snuke
Test Case 59 Input:
1 2 34 5 67 8 93-1 0 1
Output:
Snuke
Test Case 60 Input:
1 2 34 5 67 8 930 0 0
Output:
Snuke
Test Case 61 Input:
1 2 34 5 67 8 930 1 2
Output:
Snuke
Test Case 62 Input:
1 2 34 5 67 8 931 0 3
Output:
Snuke
Test Case 63 Input:
1 2 34 5 67 8 931 2 3
Output:
Snuke
Test Case 64 Input:
1 2 34 5 67 8 931 5 9
Output:
Snuke
Test Case 65 Input:
1 2 34 5 67 8 9310 11 12
Output:
Snuke
Test Case 66 Input:
1 2 34 5 67 8 9310 A 12
Output:
Snuke
Test Case 67 Input:
1 2 34 5 67 8 93A B C
Output:
Snuke
Test Case 68 Input:
1 2 34 5 67 8 93a b c
Output:
Snuke
Test Case 69 Input:
1 2 34 5 67 8 9A
Output:
Snuke
Test Case 70 Input:
1 2 34 5 67 8 9ATest Case Input:1 2 34 5 67 8 90Test Case Input:1 2 34 5 67 8 910Test Case Input:1 2 34 5 67 8 9-5Test Case Input:1 2 34 5 67 8 92 5 7 10Test Case Input:1 2 34 5 67 8 9-1 -2 -3Test Case Input:1 2 34 5 67 8 93 5 2Test Case Input:1 2 34 5 67 8 91 2 3 4
Output:
Snuke
Test Case 71 Input:
1 2 34 5 67 8 9ATest Case Input:1 2 34 5 67 8 90Test Case Input:1 2 34 5 67 8 93Test Case Input:1 2 34 5 67 8 91.5Test Case Input:1 2 34 5 67 8 910
Output:
Snuke
Test Case 72 Input:
1 2 34 5 67 8 9ATest Case Output:NumberFormatException
Output:
Snuke
Test Case 73 Input:
1 2 34 5 67 8 9A1 2 3
Output:
Snuke
Test Case 74 Input:
1 2 34 5 A7 8 926 7
Output:
Snuke
Test Case 75 Input:
1 A 34 5 67 8 931 2 3
Output:
Snuke
Test Case 76 Input:
1 A 34 B 67 C 931 2 3
Output:
Snuke
Test Case 77 Input:
1.5 2.5 3.54.5 5.5 6.57.5 8.5 9.531.5 2.5 3.5
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 78 Input:
101 21 32 42 53 63 74 84 95 10
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.LinkedList.poll()" is null
	at Main.main(Main.java:39)
Test Case 79 Input:
101 22 32 43 53 64 74 85 96 10
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.LinkedList.poll()" is null
	at Main.main(Main.java:39)
Test Case 80 Input:
21 2
Output:
Snuke
Test Case 81 Input:
21 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 261 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Output:
Snuke
Test Case 82 Input:
22-10
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 3
	at Main.main(Main.java:12)
Test Case 83 Input:
23-10
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:11)
Test Case 84 Input:
23-202
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:11)
Test Case 85 Input:
23 5 7 9 11 13 15 17 19 21 23 25 27 29 31 33 35 37 39 41 43 45 47 49 51 532 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at Main.main(Main.java:11)
Test Case 86 Input:
25 10 15 20 25 30 35 40 45 50 55 60 65 70 75 80 85 90 95 100 105 110 115 120 125 130 1351 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26PREVIOUSLY GENERATED TEST CASES
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 5 out of bounds for length 3
	at Main.main(Main.java:11)
Test Case 87 Input:
30-202
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index -2 out of bounds for length 4
	at Main.main(Main.java:12)
Test Case 88 Input:
31 21 3
Output:
Fennec
Test Case 89 Input:
31 21 3Test Case Output:Fennec
Output:
Fennec
Test Case 90 Input:
31 22 3
Output:
Fennec
Test Case 91 Input:
31 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 261 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 261 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:12)
Test Case 92 Input:
32-120
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 4
	at Main.main(Main.java:12)
Test Case 93 Input:
32-202
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index -2 out of bounds for length 4
	at Main.main(Main.java:12)
Test Case 94 Input:
321 -2 3
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index -2 out of bounds for length 4
	at Main.main(Main.java:11)
Test Case 95 Input:
32468
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:12)
Test Case 96 Input:
32 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 4 out of bounds for length 4
	at Main.main(Main.java:12)
Test Case 97 Input:
3a b c d e f g h i j k l m n o p q r s t u v w x y z1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:9)
Test Case 98 Input:
41 21 33 4
Output:
Fennec
Test Case 99 Input:
41 22 33 4
Output:
Snuke
Test Case 100 Input:
41 23 4
Error:
Exception in thread "main" java.util.NoSuchElementException
	at java.base/java.util.Scanner.throwFor(Scanner.java:945)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:9)
Test Case 101 Input:
42-305
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index -3 out of bounds for length 5
	at Main.main(Main.java:12)
Test Case 102 Input:
50 11 22 33 4
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.LinkedList.poll()" is null
	at Main.main(Main.java:39)
Test Case 103 Input:
51 21 32 42 5
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.LinkedList.poll()" is null
	at Main.main(Main.java:39)
Test Case 104 Input:
51 21 32 43 5
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.LinkedList.poll()" is null
	at Main.main(Main.java:39)
Test Case 105 Input:
51 21 32 44 5
Output:
Fennec
Test Case 106 Input:
51 21 33 43 5
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.LinkedList.poll()" is null
	at Main.main(Main.java:39)
Test Case 107 Input:
51 21 33 43 51 6(Note: This test case is designed to raise an ArrayIndexOutOfBoundsException as the program tries to access index 6 of the array, which is out of bounds.)
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.LinkedList.poll()" is null
	at Main.main(Main.java:39)
Test Case 108 Input:
51 21 33 43 55 6
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.LinkedList.poll()" is null
	at Main.main(Main.java:39)
Test Case 109 Input:
51 21 34 5Test Case Input:41 22 33 4Test Case Input:31 22 3Test Case Input:0Test Case Input:-51 21 33 43 5Test Case Input:41 21 33 4Test Case Input:71 21 32 42 53 63 7Test Case Input:31 22 3Test Case Input:-31 22 3Test Case Input:31 22 3
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:9)
Test Case 110 Input:
51 22 32 44 5
Output:
Fennec
Test Case 111 Input:
51 22 33 44 5
Output:
Fennec
Test Case 112 Input:
51 2 3 4 51 2 3 4 51 2 3 4 51 2 3 4 51 2 3 4 5
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.LinkedList.poll()" is null
	at Main.main(Main.java:39)
Test Case 113 Input:
51 2 3 4 51 2 3 4 55 4 3 2 12 3 4 5 11 1 1 1 11 1 1 1 11 1 1 1 11 1 1 1 11 1 1 1 11 1 1 1 1
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.LinkedList.poll()" is null
	at Main.main(Main.java:39)
Test Case 114 Input:
51 2 3 4 51 2 3 4 56 7 8 9 1011 12 13 14 1516 17 18 19 2021 22 23 24 25
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.LinkedList.poll()" is null
	at Main.main(Main.java:39)
Test Case 115 Input:
52-2042
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index -2 out of bounds for length 6
	at Main.main(Main.java:12)
Test Case 116 Input:
53-2-31-54
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index -2 out of bounds for length 6
	at Main.main(Main.java:12)
Test Case 117 Input:
53-240-13
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index -2 out of bounds for length 6
	at Main.main(Main.java:12)
Test Case 118 Input:
53-240-31
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index -2 out of bounds for length 6
	at Main.main(Main.java:12)
Test Case 119 Input:
531-23-40
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index -2 out of bounds for length 6
	at Main.main(Main.java:11)
Test Case 120 Input:
531-23-45
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index -2 out of bounds for length 6
	at Main.main(Main.java:11)
Test Case 121 Input:
531 -2 3 -4 5
Error:
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index -2 out of bounds for length 6
	at Main.main(Main.java:11)
Test Case 122 Input:
61 21 31 42 52 6
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.LinkedList.poll()" is null
	at Main.main(Main.java:39)
Test Case 123 Input:
61 21 32 42 53 6
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.LinkedList.poll()" is null
	at Main.main(Main.java:39)
Test Case 124 Input:
61 21 32 42 54 6
Output:
Fennec
Test Case 125 Input:
61 22 31 44 55 6
Output:
Fennec
Test Case 126 Input:
61 22 32 43 54 6
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.LinkedList.poll()" is null
	at Main.main(Main.java:39)
Test Case 127 Input:
71 21 32 42 53 63 7
Error:
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.lang.Integer.intValue()" because the return value of "java.util.LinkedList.poll()" is null
	at Main.main(Main.java:39)
Test Case 128 Input:
81 21 32 42 53 63 74 8Explanation: This test case creates a scenario where the input graph has 8 vertices and 7 edges.
Output:
Fennec
Test Case 129 Input:
A B CD E FG H I3J K L
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 130 Input:
a2-202
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 131 Input:
a 2 34 5 67 8 931 2 3
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 132 Input:
a b cd e fg h i2j k l
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 133 Input:
a b cd e fg h i31 2 3
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 134 Input:
a b cd e fg h i3j k l
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 135 Input:
a b cd e fg h i3x y z
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
Test Case 136 Input:
null2 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 262 4 6 8 10 12 14 16 18 20 22 24 26 28 30 32 34 36 38 40 42 44 46 48 50 521 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26
Error:
Exception in thread "main" java.util.InputMismatchException
	at java.base/java.util.Scanner.throwFor(Scanner.java:947)
	at java.base/java.util.Scanner.next(Scanner.java:1602)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2267)
	at java.base/java.util.Scanner.nextInt(Scanner.java:2221)
	at Main.main(Main.java:5)
